components/Common/Dropdown/index.tsx

```diff
@@ -8,10 +8,11 @@ type DropdownProps = {
   items: Array<DropdownItem>;
   shouldShow: boolean;
   styles: CSSProperties;
+  children: React.ReactNode;
```

Comment:
Instead of declaring children here, please wrap `DropdownProps = PropsWithChildren<{ .... }>`